### ISSUE TITLE
Performance statistics @ iOS

### ISSUE THREAD

**Issue by Lev-Guzeev** [mikefogg/CrittercismModule#2]
Hey Mike
The statistics (https://app.crittercism.com/developers/network/<app_id>) for iOS is empty on Crittercism, however crashes and app loads are registered.
Thanks for your efforts!

**Comment by mikefogg** [comment]
@Lev-Guzeev, hmm... completely empty? Right now in mine (using the iOS module) I'm seeing Device, OS, and Carrier trends. Is that what you'd expect to see?

**Comment by Lev-Guzeev** [comment]
Exactly there's everything zero, and also Troubleshoot->App Performance section - it only contains network activity of another analytics tool I have in app - Flurry, but no backend calls, however they are in place for Android.

**Comment by mikefogg** [comment]
I'm going to guess this has nothing to do with it, but I wanted to make sure, did you ever upload a DSYM file for the build you're using in the App Settings section?

**Comment by Lev-Guzeev** [comment]
Yes, I've uploaded dSYM file

**Comment by stephenwoodford** [comment]
I'm seeing the performance statistics fine on my end using the iOS module.

Is it possible Flurry is conflicting with this module as well? I would try removing it and see if you get any data.

**Comment by mikefogg** [comment]
Yeah, it's possible that Flurry is intercepting the network calls before it gets through to Crittercism. Can you try removing that and see if that helps?

There may be a slight delay even once it does start reporting before you see those start rolling in. For me I noticed it took at least a day to see that first round of reporting, but, that could have just been me.

**Comment by mikefogg** [comment]
@Lev-Guzeev, are you still seeing an issue with Flurry and New Relic disabled?

**Comment by Lev-Guzeev** [comment]
Mike, seems it really takes some time to display statistics on site, so it seems to be working with flurry as well :)

**Comment by hardikamal** [comment]
Me to facing same problem...any solution found???
